data/participants.json

```diff
@@ -545,6 +546,7 @@
   },
   {
 
```

Comment:
Every difference other than adding my own info comes when i am resolving the conflicts.